_pages/speaking-resources.md

```diff
@@ -26,21 +28,42 @@ Presenters, regardless of experience, sometimes want a little help. If you’d l
 * [Dr. Russell Keith-Magee](mailto:russell@keith-magee.com), Founder of the BeeWare project, developing GUI tools and libraries to support the development of Python software on desktop and mobile platforms. He is also a 13 year veteran of the Django Core Team, and for 5 years, was President of the Django Software Foundation. In his day job, he wrangles data pipelines for Survata.
 {% endcomment %}
 
+## Talk Guidelines
+
+### Accessibility
+
+* You can submit a draft copy of your slides with notes indicating what you plan to say a minimum of three weeks before your talk to give the captioners time to train on your data and therefore improve caption accuracy. This is extremely helpful to our attendees who read the live captions if you're using jargon beyond "Django" and "the ORM".
+* These slides will not be shared outside of the organizing team and the captioners. Send the slides to hello AT djangocon DOT us when you are ready. It's perfectly fine if your slides and notes change between the draft and final version.
+* When describing media in your talk (images, screenshots, graphs, videos, etc.), please give a verbal description of what you're showing so that blind or vision-impaired attendees can understand what you're describing.
+  * Example: instead of "here's a graph showing performance improvements with our new async code," say "here's a line graph showing our average response time that decreased from 450 milliseconds to 150 milliseconds after adding our new async code."
+
+### General
+
+* Use gender-neutral language wherever possible. Don't default to using "he" or "him" everywhere.
+* Be respectful of other speakers, your audience, and the community at large, especially when offering critiques. It's possible to say "this needs improvement" without saying "this is bad."
+* Consider including your social media handle(s) on your opening and closing slides! If you have space, you can even include them on the footer of every slide.
+* Consider publishing your slides after your talk (on the platform of your choice) and sharing the link with the #djangocon hashtag on social media.
```

Comment:
Can we use camel casing on the hashtags? It's preferred on Mastodon.

Reply:
Definitely